integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java

```diff
@@ -72,7 +70,12 @@
         defineInvalidIntrospectionMockTokenStubForUserWithRoles("expired", emptySet());
 
         LOG.infof("Keycloak started in mock mode: %s", server.baseUrl());
-        return Collections.singletonMap("quarkus.oidc.auth-server-url", server.baseUrl() + "/auth/realms/quarkus");
+        Map<String, String> conf = new HashMap<>();
+        conf.put("quarkus.oidc.auth-server-url", server.baseUrl() + "/auth/realms/quarkus");
+        conf.put("quarkus.oidc.code-flow.auth-server-url", server.baseUrl() + "/auth/realms/quarkus");
+        conf.put("keycloak-url", server.baseUrl());
```

Comment:
@cemnura now that you have `keycloak-url`, we don't need the other 2 properties, right ?

Reply:
@cemnura Never mind, looks like both are needed :-), confirmed in the unrelated test